Documentation/compatibility/winforms-accessibility-changes-472.md

```diff
@@ -0,0 +1,107 @@
+## Accessibility improvements in Windows Forms controls for .NET 4.7.2
+
+### Scope
+Major
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Windows Forms Framework is improving how it works with accessibility technologies to better support Windows Forms customers. These include the following changes:
+- Changes to improve display during High Contrast mode.
+- Changes to improve the keyboard navigation in the DataGridView and MenuStrip controls. 
+- Changes to interaction with Narrator.
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+__How to opt in or out of these changes__
+  
+In order for the application to benefit from these changes, it must run on the .NET Framework 4.7.2 or later. The application can benefit from these changes in either of the following ways:
+- It is recompiled to target the .NET Framework 4.7.2. These accessibility changes are enabled by default on Windows Forms applications that target the .NET Framework 4.7.2 or later.
+- It opts out of the legacy accessibility behaviors by adding the following [AppContext Switch](https://docs.microsoft.com/dotnet/framework/configure-apps/file-schema/runtime/appcontextswitchoverrides-element) to the ```<runtime>``` section of the app config file and setting it to false, as the following example shows.
+```
+    <?xml version="1.0" encoding="utf-8"?>
+    <configuration>
+      <startup>
+        <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.7"/>
+      </startup>
+      <runtime>
+        <!-- AppContextSwitchOverrides value attribute is in the form of 'key1=true|false;key2=true|false  -->
+        <AppContextSwitchOverrides value="Switch.UseLegacyAccessibilityFeatures=false;Switch.UseLegacyAccessibilityFeatures.2=false" />
+      </runtime>
+    </configuration>
+```
+Applications that target the .NET Framework 4.7.2 or later and want to preserve the legacy accessibility behavior can opt in to the use of legacy accessibility features by explicitly setting this AppContext switch to ```true```.
+
+__Use of OS-defined colors in High Contrast themes__
+- The drop down arrow arrow of the `T:System.Windows.Forms.ToolStripDropDownButton` now has uses now use OS-defined colors in High Contrast theme.
+- `T:System.Windows.Forms.Button`, `T:System.Windows.Forms.RadioButton` and `T:System.Windows.Forms.CheckBox` controls with `P:System.Windows.Forms.Control.FlatStyle` set to <xref:System.Windows.Forms.FlatStyle.Flat?displayProperty=nameWithType> or <xref:System.Windows.Forms.FlatStyle.Popup?displayProperty=nameWithType> now use OS-defined colors in High Contrast theme when selected. Previously, text and background colors were not contrasting and were hard to read.
+- Controls contained within a `System.Windows.Forms.GroupBox` which has its `P:System.Windows.Forms.Enabled` property set to _false_ will now use OS-defined colors in High Contrast theme.

```

Comment:
which --> that
false is better fenced (no risk of it being localized)
The Enabled link won't resolve since there's no type. It should be \<xref:System.Windows.Forms.Control.Enabled>